textBasedAdventure.js

```diff
@@ -1,5 +1,139 @@
 const readline = require('readline-sync')
+// console.log(typeof Number(number))
+
 
 let nameInput = readline.question("Enter your name: ")
+console.log(`Hello ${nameInput}!  May the odds be ever in your favor. Let the games begin ! `)
+
+let age = readline.question("How old are you?  ")
+    if (age >= 0) {
+        console.log("You’re never too young for the game  "); 
+
+} 
+const district= readline.question("pick a district from 1 - 3  ") 
+switch (`${district}`) {
```

Comment:
you're switch doesn't work because you pass ``${district}`` as an argument instead of `district`